tasks/main.yml

```diff
@@ -50,6 +59,14 @@
   when: "{{ freshclam_enabled }}"
   tags: usb
 
+- name: Set cron job for scanning / at 12:00 on a Wednesday
```

Comment:
Is there another way to do this in the ClamAV conf instead of a cron job? It'd be preferable to do it there if possible. That's where the pentesters checked for a recurring job.